app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
Comments are an important part of the code, but in order for them to be helpful they should be meaningful and provide context and extra information that cannot be directly infered by the code it relates to. In this case, I think comments do not provide too much of a valuable information so maybe we can remove them. What do you think?